great_expectations/render/renderer/page_renderer.py

```diff
@@ -152,7 +152,9 @@ def render(
     def _parse_run_values(
         self, validation_results: ExpectationSuiteValidationResult
     ) -> Tuple[str, str]:
-        run_id: Union[str, dict, RunIdentifier] = validation_results.meta["run_id"]
+        run_id: Union[str, dict, RunIdentifier] = (
+            validation_results.run_id or validation_results.meta.get("run_id", {})
+        )
```

Comment:
I'd like to change these types of instances to just use `validation_results.run_id` but let's do this incrementally.